ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -9,7 +9,8 @@
     </p>
 
     <p v-if="isCancelUnitNote" class="mb-15" data-test-id="cancel-note-info">
-      <b>Note:</b> Once this Cancel Note is registered, the original Notice of Caution will no longer
+      <b>Note:</b> Once this Cancel Note is registered, the original
+      {{ getCancelledUnitNoteHeader().replace(/[()]/g,"") }} will no longer
```

Comment:
You can get the unit note info from `unitNotes.ts` under the `resources` folder. 
E.g. `UnitNotesInfo[doc_type_here].header`

Reply:
I have made some modification, please review again, thank you